pkgs/development/libraries/cxxtools/default.nix

```diff
@@ -1,23 +1,32 @@
-{ lib, stdenv, fetchurl }:
+{ lib, stdenv
+, fetchFromGitHub
+, autoreconfHook
+, openssl
+}:
 
 stdenv.mkDerivation rec {
-  version = "2.2.1";
   pname = "cxxtools";
+  version = "unstable-2021-12-23";
 
-  src = fetchurl {
-    url = "http://www.tntnet.org/download/${pname}-${version}.tar.gz";
-    sha256 = "0hp3qkyhidxkdf8qgkwrnqq5bpahink55mf0yz23rjd7rpbbdswc";
+  src = fetchFromGitHub {
+    owner = "maekitalo";
+    repo = "cxxtools";
+    rev = "31a212fe400b36cc5b9ea0dd76d9b5facfde914d";
+    hash = "sha256-h7/G5aPnEsGjOL2NrKAnw4VfWTr8o4u574albdqFdjg=";
   };
 
+  nativeBuildInputs = [ autoreconfHook ];
+  buildInputs = [ openssl.dev ];
+
   configureFlags = lib.optional stdenv.isAarch64 "--with-atomictype=pthread";
```

Comment:
I guess we don't need it anymore, see https://github.com/maekitalo/cxxtools/commit/c5baa0b7756ee88f51ec58fe9a4d3b416266cc9f